code/_onclick/hud/new_player.dm

```diff
@@ -15,6 +16,8 @@
 	layer = LOBBY_BUTTON_LAYER
 	screen_loc = "TOP,CENTER"
 
+/atom/movable/screen/lobby/proc/SlowInit()
```

Comment:
Can you doc-comment this?

Also add a `return` here just so it doesn't look out of place as being an empty proc.